### DISCUSSION TITLE
enabl touchpad in qtile

### DISCUSSION THREAD

**Discussion by notwidow** [qtile/qtile#3889] [Q&A]
How to tap to click with touchpad in qtile???

**Comment by ShiNoNeko47** [comment]
Xorg or wayland?

**Comment by notwidow** [comment]
xorg

**Comment by ShiNoNeko47** [comment]
Add `Option "Tapping" "True"` to `/usr/share/X11/xorg.conf.d/40-libinput.conf` and restart xorg or reboot

```
...
Section "InputClass"
        Identifier "libinput touchpad catchall"
        MatchIsTouchpad "on"
        MatchDevicePath "/dev/input/event*"
        Driver "libinput"
        ...
	Option "Tapping" "True"
        ...
EndSection
...
```

**Reply by notwidow** [reply]
> Add `Option "Tapping" "True"` to `/usr/share/X11/xorg.conf.d/40-libinput.conf` and restart xorg or reboot
> 
> ```
> ...
> Section "InputClass"
>         Identifier "libinput touchpad catchall"
>         MatchIsTouchpad "on"
>         MatchDevicePath "/dev/input/event*"
>         Driver "libinput"
>         ...
> 	Option "Tapping" "True"
>         ...
> EndSection
> ...
> ```

thank you and what about wayland

**Reply by notwidow** [reply]
> Add `Option "Tapping" "True"` to `/usr/share/X11/xorg.conf.d/40-libinput.conf` and restart xorg or reboot
> 
> ```
> ...
> Section "InputClass"
>         Identifier "libinput touchpad catchall"
>         MatchIsTouchpad "on"
>         MatchDevicePath "/dev/input/event*"
>         Driver "libinput"
>         ...
> 	Option "Tapping" "True"
>         ...
> EndSection
> ...
> ```

amd how to set wallpaper in qtile

**Reply by Tokariew** [reply]
either by using built in option when defing screens https://docs.qtile.org/en/stable/manual/config/screens.html?highlight=screen#reference or using other programs like feh or xwallpaper
